### PR TITLE
One tx common deployment

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: schemesVoteParams/FundingRequestParams.json [hunk 1]
```diff
@@ -10,6 +10,5 @@
     "thresholdConst": 2000,
     "voteOnBehalf": "0x0000000000000000000000000000000000000000",
     "votersReputationLossRatio": 1,
-    "activationTime": 0,
-    "paramsHash": "0x62f2af6100a7374bcea52cbc2b654eb4dce056d52ea41eda08aea55e1a871ee2"
+    "activationTime": 0
```
Comment (orenyodfat) [inline schemesVoteParams/FundingRequestParams.json:13]: this should be set to the fundingDeadLine ?!

File: schemesVoteParams/JoinAndQuitParams.json [hunk 2]
```diff
@@ -10,6 +10,5 @@
     "thresholdConst": 2000,
```
Comment (orenyodfat) [inline schemesVoteParams/JoinAndQuitParams.json:10]: this param is already agreed by the common team ? @jellegerbrandy @leviadam ?

File: index.js [hunk 3]
```diff
@@ -65,7 +64,7 @@ function getSetSchemesData({
     }
 
     const joinAndQuitArgs = Object.values({
-        avatar,
+        avatar: '0x0000000000000000000000000000000000000000',
```
Comment (orenyodfat) [inline index.js:67]: can this be replaced with a Constant : NULL_ADDRESS ?

File: index.js [hunk 4]
```diff
@@ -135,7 +134,7 @@ function getSetSchemesData({
     });
 
     const dictatorArgs = Object.values({
-        avatar,
+        avatar: '0x0000000000000000000000000000000000000000',
         owner: '0xbBb06cD354D7f4e67677f090eCc3f6E5916E2447'
```
Comment (orenyodfat) [inline index.js:138]: lets make this as a Constant ..maybe in a common_constant param file (maybe json)

File: schemesVoteParams/FundingRequestParams.json (hunk 1 above)
Comment (ben-kaufman) [reply schemesVoteParams/FundingRequestParams.json:13]: Jelle wanted it to be done a bit differently, see in the code

File: index.js (hunk 4 above)
Comment (ben-kaufman) [reply index.js:138]: I can just make it a constant in the file, why create a new file just for that?

Comment (orenyodfat) [reply index.js:140]: ok

File: schemesVoteParams/FundingRequestParams.json (hunk 1 above)
Comment (orenyodfat) [reply schemesVoteParams/FundingRequestParams.json:13]: @jellegerbrandy ?

Comment (ben-kaufman) [reply schemesVoteParams/FundingRequestParams.json:13]: Right

**Pull request merged**